docs_app/content/tutorial/3-creating-an-observable.md

```diff
@@ -0,0 +1,140 @@
+# 3. Creating An Observable
+
+## TLDR:
+
+- You should probably use a provided [creation function](API).
```

Comment:
These TLDR sections are great!

Consider re-wording the first two items to be more clear on the recommendation:

- "Use a provided creation function to create an Observable"
- "Only create an Observable with the `new Observable` constructor if no creation function will suffice"